services/callback/src/test/java/app/coronawarn/server/services/callback/controller/ApiExceptionHandlerTest.java

```diff
@@ -28,7 +28,7 @@
 
   @BeforeEach
   public void setup() {
-    initMocks(this);
+    openMocks(this);
```

Comment:
We should remove RunWith (JUnit 4) and use instead ExtendWith(MockitoExtension.class,SpringExtension.class) (JUnit 5).
If so then we can:
-  remove the openMocks/initMocks.
- remove the initialization of the callbackController by using InjectMock